docs/readthedocs/models/models.rst

```diff
@@ -48,6 +48,31 @@ Harmonized Units System
   Same applies for interpreting the obtained results.
   In all models physical values are transferred to standard units on instantiation.
 
+Equality Checks
+  To represent quantities in the models within an acceptable accuracy, the JSR 385 reference implementation
+  `Indriya <https://github.com/unitsofmeasurement/indriya>`_ is used. Comparing quantity objects or objects holding quantity
+  instances is not as trivial as it might seem, because there might be different understandings about the equality of
+  quantities (e.g. there is a big difference between two instances being equal or equivalent). After long discussions how to
+  treat quantities in the entity :code:`equals()` method, we agreed on the following rules to be applied:
+    - equality check is done by calling :code:`Objects.equals(<UnitInstanceA>, <UnitInstanceB>)` or
+      :code:`<UnitInstanceA>.equals(<UnitInstanceB>)`.
+      Using :code:`Objects.equals(<UnitInstanceA>, <UnitInstanceB>)` is necessary especially for time series data.
+      As in contrast to all other places, quantity time series from real world data sometimes are not complete and
+      hence contain missing values. To represent missing values this is the only place where the usage of :code:`null`
+      is a valid choice and hence needs to be treated accordingly. Please remember that his is only allowed in very few
+      places and you should try to avoid using :code:`null` for quantities or any other constructor parameter whenever possible!
+    - equality is given if, and only if, the quantities value object and unit are exactly equal. Value objects can become
+      e.g. :code:`BigDecimal` or :code:`Number` instances. It is important, that the object type is also the same, otherwise
+      the entities :code:`equals()` method returns false. This behavior is in sync with the equals implementation
+      of the indriya library. Hence, you should ensure that your code always pass in the same kind of a quantity instance
+      with the same underlying number format and type.
+    - for now, there is no default way to compare entities in a 'number equality' way provided. E.g. a line with a length
+      of 1km compared to a line with a length of 1000m is actually of the same length, but calling :code:`LineA.equals(LineB)`
+      would return :code:`false` as the equality check does NOT convert units. If you want to compare two entity instances
+      based on their equality you have (for now) check for each quantity manually using their :code:`isEquivalentTo()`
+      method. If you think you would benefit from a standard method that allows entity equivalence check, please consider
+      handing in an issue `here <https://github.com/ie3-institute/PowerSystemDataModel/issues>`_.
+
```

Comment:
Is it advisory to spend a word on comparison with tolerance?

Reply:
yep - will add